content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -193,12 +197,16 @@ to clean up associated resources.
 
 ### Unreserve
```

Comment:
Technically, `Unreserve` is not an extension point anymore. I'd suggest moving (and may need some rephrasing) this section as a bullet point of `Reserve`. So there can be two bullet points in `Reserve`, explaining the implementation/execution of `Reserve` and `Reserve` accordingly, under different circumstances.

Reply:
Maybe we can call them "calls" of the "Reserve extension point"

Reply:
Unreserve isn't a plugin anymore, but isn't it still an extension point? It's just that one plugin triggers two extension points now. IMO, Reserve has no special meaning or difference compared to Unreserve; so they're both extension points or they're both not. Also, users can choose to leave the Reserve implementation empty. I think having a separate Unreserve section still makes sense, but I'd like to hear your thoughts.

Reply:
I'd read the term "extension point" as an API defined in https://github.com/kubernetes/kubernetes/blob/bc60bdaded587ef638c165c49680728a57326f29/staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go#L159, and also it should be 1:1 mapped to the rectangles in the architecture picture. Specifically, a plugin can implement multiple extension points, and an extension point represents a phase in a Pod's scheduling workflow. 

Most "extension points" just have one method defined, such as Fitler/Score. We do also have other ones such as PreFilter which has method "PreFilter" as well as "AddPod"/"RemovePod", implicitly though.

Reply:
so, UnReserve is a scheduling phase, but not an extension point.

Reply:
> UnReserve is a scheduling phase, but not an extension point.

I prefer to read it this way, and it's a _dynamic_ phase, that's why categorized it with a _fixed_ phase "Reserve" in pairs.

Reply:
>I'd read the term "extension point" as an API defined in https://github.com/kubernetes/kubernetes/blob/bc60bdaded587ef638c165c49680728a57326f29/staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go#L159

Fair enough, that's a good point.

Done.